devenv.nix

```diff
@@ -18,9 +18,12 @@
     pipenv install --dev
     npm i
     gulp migrate
-    gulp
```

Comment:
This change is necessary for the vscode debugger to have a chance to start django itself (rather than via gulp). The caveat is the extra step of running `gulp` in devcontainers or `devenv up` instead (thus the change to the devenv docs)